memebox-streamdeck/scripts/build.js

```diff
@@ -49,7 +48,7 @@ measureFileSizesBeforeBuild(paths.appBuild)
 		return build(previousFileSizes);
 	})
 	.then(
-		({ stats, previousFileSizes, warnings }) => {
```

Comment:
maybe rather use a comment version of  /* stats, previousFileSizes */ so that IF needed to change something atleast everyone would know that those exist

Reply:
Done!